compiler_tester/src/vm/eravm/mod.rs

```diff
@@ -49,6 +49,8 @@ pub struct EraVM {
     published_evm_bytecodes: HashMap<web3::types::U256, Vec<web3::types::U256>>,
     /// The storage state.
     storage: HashMap<zkevm_tester::runners::compiler_tests::StorageKey, web3::types::H256>,
+    /// The current EVM block number
```

Comment:
It is used for EraVM as well, isn't it?

Reply:
It's currently not used for the EraVM target, if I'm not mistaken that would break some tests as some of them actively rely on the block number always being `CURRENT_BLOCK_NUMBER_ERAVM` (`300`).